jcl/jpp_configuration.xml

```diff
@@ -326,4 +326,12 @@
 		<source path="src"/>
 	</configuration>
 
+	<!--
+	This configuration is intended only for use by JDK8 extension repo:
+	it is not expected to usable for generating an eclipse project.
+	-->
+	<configuration label="SIDECAR18-SE-OPENJ9-CLOSED" outputpath="outputpath-not-used">
+		<source path="src/share/classes"/>
+	</configuration>
```

Comment:
I don't think we need both `DDR_VM` and this configuration.
I suggest we adjust this so it can subsume the current uses of `DDR_VM`.
In which case the label should be more generic (e.g. 'BASIC', 'GENERIC', 'MINIMAL' or something else appropriate).
The source path should just be `src` (so it can be used in the current `DDR_VM` cases); the use in jdk8 can add 'share/classes' to the output directory.

Reply:
> The source path should just be src (so it can be used in the current DDR_VM cases); the use in jdk8 can add 'share/classes' to the output directory.

The initial issue I had was to include the java files under `openj9-openjdk-jdk8/closed/adds/jdk/src/share/classes` which was organized differently from `JDK11+` extension repos like `src/java.base/share/classes` specified at `SIDECAR18-SE-OPENJ9` `source path`. Not sure adding 'share/classes' to the output directory helps.

Reply:
Or the source path could just be `"."`.

Reply:
How about following change?
from
```
	<configuration label="DDR_VM" outputpath="outputpath-not-used">
		<source path="src"/>
	</configuration>
```
to
```
	<configuration label="GENERIC" outputpath="outputpath-not-used">
		<source path="src"/>
		<source path="src/share/classes"/>
	</configuration>
```
This can serve both original configuration `DDR_VM` and new JDK8 closed addition. 
All the extension repo references to the configuration `DDR_VM` will have to be updated.

Reply:
With both `src` and `src/share/classes`, we'll get duplication of the whole `share` subtree.

This seems to work:
```
	<configuration label="GENERIC" outputpath="outputpath-not-used">
		<source path="."/>
	</configuration>
```
Yes, all extension repos will need to change when `DDR_VM` is removed.
The new use of `GENERIC` in jdk8 would be
```
	$(call RunJPP, GENERIC, $(TOPDIR)/closed/adds/jdk/src/share/classes, $(JPP_DEST)) \
		$(IncludeIfUnsure)
```
and the use of `DDR_VM` would be replaced by
```
	$(call RunJPP, GENERIC, $(OPENJ9_TOPDIR)/debugtools/DDR_VM/src, $(J9JCL_SOURCES_DIR)/ddr) \
		$(IncludeIfUnsure) \
		-macro:define "JAVA_SPEC_VERSION=8"
```

Reply:
`<source path="."/>` along w/ `$(TOPDIR)/closed/adds/jdk/src/share/classes` works.
Updated the PR and JDK8 extension repo one, pls have another look.

Other extension repo PRs adopting `GENERIC` configuration to be created shortly.